src/index.ts

```diff
@@ -113,7 +113,7 @@ export class Podcast {
     });
 
     customElements.push({
-      "itunes:explicit": !!options.itunesExplicit,
+      "itunes:explicit": typeof options.itunesExplicit === "boolean" ? !!options.itunesExplicit : options.itunesExplicit,
```

Comment:
I think we should not support `true` or `false` anymore and instead convert `true` to `"yes"` and `false` to `"no"`

Reply:
Two reasons.
Primary: the official documentation says true|false
The other values suggested by the W3C validator are referring to old Apple specs, back in 2008
https://web.archive.org/web/20080726084241/http://www.apple.com/itunes/store/podcaststechspecs.html#explicit
I used two other validators, suggested by Apple here https://help.apple.com/itc/podcasts_connect/#/itcbaf351599, and both validators accept true|false|"yes"|"no"|"clean".
Secondary: no breaking changes.